public/resources/fontello-24c5e6ad/demo.html

```diff
@@ -0,0 +1,311 @@
+<!DOCTYPE html>
```

Comment:
thanks for pointing that out. from the readme:  "If your project is open-source, usually, it will be ok to make LICENSE.txt file publicly available in your repository." Se we can remove the readme file, but we need to keep the license file around somehwere.
As for the other files, I believe it is all CSS directly affecting the icons, where else would you propose we put it?